src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,258 @@
+package Duke;

```

Comment:
Good job overall, can see a lot of effort put in your work!👍 I like the implantation of the logic.  A little bit of checking and following coding standards would be good.

Reply:
Thanks for looking through my codes! :smiley: